README.md

```diff
@@ -85,7 +85,7 @@ After you're finished with this demo, clean up your resources.
 ![Image of disassociate option](images/disassociate.png)
 
 ## Learn more
-
+Test change
```

Comment:
The `is` operator performs an identity comparison that uses the built-in `is()` method to test if the compared items are the same object or  not. It should be used with only singletons, such as `None`. We recommend that you use the `==` or `!=` equality operators to determine if two objects are equal.

[Learn more](https://docs.python.org/3/reference/datamodel.html#object.eq)